packages/react/src/components/DataTable/TableExpandHeader.js

```diff
@@ -57,23 +55,12 @@ TableExpandHeader.propTypes = {
    * Specify the string read by a voice reader when the expand trigger is
    * focused
    */
-  ariaLabel: PropTypes.oneOfType([
-    requiredIfGivenPropIsTruthy('enableExpando', PropTypes.string),
-    requiredIfGivenPropIsTruthy('enableToggle', PropTypes.string),
-  ]),
+  ariaLabel: requiredIfGivenPropIsTruthy('enableToggle', PropTypes.string),
 
   children: PropTypes.node,
 
   className: PropTypes.string,
 
-  /**
-   * The enableExpando prop is being replaced by enableToggle
-   */
-  enableExpando: deprecate(
```

Comment:
I think this was also deprecated as part of v11 changes. https://github.com/carbon-design-system/carbon/issues/10317